src/AdminTab/TableList.tsx

```diff
@@ -40,10 +34,9 @@ type TableDropdownProps = {
   showMetabaseHint: boolean;
 };
 
-const TableDropdown: FunctionComponent<TableDropdownProps> = ({ table, onOpenMetabaseTab, showMetabaseHint }) => {
+const TableActions: FunctionComponent<TableDropdownProps> = ({ table, onOpenMetabaseTab }) => {
   const t = useT('AdminTab::TableList::TableDropdown');
 
-  const [metabaseHintHovered, setMetabaseHintHovered] = useState(false);
```

Comment:
I think some form of hint should be retained, as it was specifically requested before.

You can highlight (using `type=primary`) the two metabase buttons, and apply the hint tooltip to the leftmost one (or the enclosing `Space`), to not conflict with the others (it should not conflict, as it is placed differently).

Reply:
> to not conflict with the others

The tooltips are above, the popconfirm is below, so I suppose we place the hint left from all the buttons (parent of `<Space>`). I'll try this out.

Reply:
The hint disappears on hover, so it's likely it will be gone by the time either of the others comes up.

Reply:
Oh man I was scratching my head why it isn't working. I thought the hint triggers on import and not a one-time thing that is persisted.